src/actions/onboardingActions.js

```diff
@@ -112,6 +115,7 @@ export const setupUserAction = (username: ?string, recoveryData?: Object) => {
     const privateKey = wallet?.privateKey;
     if (!privateKey) {
       reportLog('setupUserAction failed: no privateKey', { recoveryData });
+      log.info('setupUserAction failed: no privateKey', { recoveryData });
```

Comment:
I wonder if it makes sens to call log.info after `reportLog` as `reportLog` is actually making sentry capture the message (i.e. not only a breadcrumb but actual event capture). This comment applies to all instances of `log.info` + `reportLog` and also similar cases of `log.error` + `reportErrorLog`.